local_store.go

```diff
@@ -11,10 +11,22 @@ import (
 
 	"github.com/theupdateframework/go-tuf/data"
 	"github.com/theupdateframework/go-tuf/encrypted"
-	"github.com/theupdateframework/go-tuf/sign"
+	"github.com/theupdateframework/go-tuf/pkg/keys"
 	"github.com/theupdateframework/go-tuf/util"
 )
 
+func signers(privateKeys []*data.PrivateKey) []keys.Signer {
+	res := make([]keys.Signer, len(privateKeys))
+	for i, k := range privateKeys {
+		signer, err := keys.GetSigner(k)
+		if err != nil {
+			continue
```

Comment:
This only appears in a file system store, but I resolved this so that the capacity is len(privateKeys) but the array is initialized with size 0 and we append. So the returned array only contains valid signers